estate/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+{
+    'name': "Real Estate",
+    'version': '1.0',
+    'sequence' : 1,
+    'depends': ['base'],
+    'author': "ATPA",
+    'category': 'Category',
+    'description': """
+    Description text
+    """,
+    'data' : ['security/ir.model.access.csv',
+    'views/estate_property_views.xml',
+    'views/estate_menus.xml'],
```

Comment:
```suggestion
    'data': [
        'security/ir.model.access.csv',
        'views/estate_property_views.xml',
        'views/estate_menus.xml'
    ],
```
Make sure to follow coding guidelines.